services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/config/DefaultConfig/DefaultConfigRepository.java

```diff
@@ -0,0 +1,18 @@
+
+package us.dot.its.jpo.ode.api.accessors.config.DefaultConfig;
+
+import java.util.List;
+
+import org.springframework.data.mongodb.core.query.Query;
+import us.dot.its.jpo.conflictmonitor.monitor.models.config.DefaultConfig;
+
+public interface DefaultConfigRepository {
+    Query getQuery(String key);
+
+    long getQueryResultCount(Query query);
+
+    List<DefaultConfig> find(Query query);
```

Comment:
I'm not sure it needs addressed in this release, but I see a similar warnings in several places about `xxx is a raw type. References to generic type xxx<T> should be parameterized Java(16777788)`

Reply:
These are now resolved - I had to suppress 2 warnings in default_config and intersection_config. I have not been able to find a resolution to the remaining warnings without unnecessarily re-factoring the code

Reply:
Backlog item for future iteration then?

Reply:
Created a work item: https://dev.azure.com/SOC-OIT/CDOT/_backlogs/backlog/CDOT%20Connected%20Vehicles/Backlog%20items/?workitem=289205